### DISCUSSION TITLE
Can I use LLAVA Large Language and Vision Assistan with koboldcpp

### DISCUSSION THREAD

**Discussion by AB0x** [LostRuins/koboldcpp#636] [Q&A]
There is this https://huggingface.co/jartine/llava-v1.5-7B-GGUF/tree/main gguf model, can i interface with koboldcpp and upload images?

**Comment by kalle07** [comment]
those models you can use in texgen-web-ui (oobadooga)
but its not better than blip2-coco

**Reply by LostRuins** [reply]
Support is desirable but it is not a high priority right now
